[Merged by Bors] - Update Lighthouse book and some FAQs (#4178)
sigp/lighthouse: book/src/installation.md

- [Docker images](./docker.md).
- [Building from source](./installation-source.md).

Additionally, there are two extra guides for specific uses:

- [Raspberry Pi 4 guide](./pi.md).
- [Cross-compiling guide for developers](./cross-compiling.md).

There are also community-maintained installation methods:

- [Homebrew package](./homebrew.md).
- Arch Linux AUR packages: [source](https://aur.archlinux.org/packages/lighthouse-ethereum),
  [binary](https://aur.archlinux.org/packages/lighthouse-ethereum-bin).

## Recommended System Requirements

> Before [The Merge](https://ethereum.org/en/roadmap/merge/), Lighthouse is able to run on its own with low to mid-range consumer hardware, but will perform best when provided with ample system resources. 

Reviewer: ```suggestion
Before [The Merge](https://ethereum.org/en/roadmap/merge/), Lighthouse was able to run on its own with low to mid-range consumer hardware, but would perform best when provided with ample system resources. 
```
